contracts/StreamManager.sol

```diff
@@ -201,25 +173,24 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
     ///@dev payee can claim tokens which is proportional to elapsed time (exactly seconds).
     function claim()
         onlyPayee
-        notTerminatedInCliffPeriod
         onlyAfterCliffPeriod
         nonReentrant
         external
     {
         uint256 claimedAt = block.timestamp;
-        uint256 terminatedAt = streamInstances[msg.sender].terminatedAt;
-        uint256 lastClaimedAt = streamInstances[msg.sender].lastClaimedAt;
-        address token = streamInstances[msg.sender].token;
-        uint256 terminationPeriod = streamInstances[msg.sender].terminationPeriod;
-        bool isTerminated = streamInstances[msg.sender].isTerminated;
+        OpenStream storage streamInstance = streamInstances[msg.sender];
```

Comment:
@accompany1205 use that in all functions

Reply:
Fixed